components/QuestionWrapper.tsx

```diff
@@ -0,0 +1,11 @@
+import styles from '../styles/QuestionWrapper.module.css'
+
+type Props = { children?: JSX.Element }
+
+/**
+ * This wrapper ensures that only one question is shown at a time.
+ */
+const QuestionWrapper = ({ children }: Props) => (
+  <section className={styles.q}>{children}</section>
```

Comment:
Based on https://github.com/ibi-group/survey/pull/1#discussion_r872844124 and its thread, I recommend surrounding `children` with `<form>`.

Reply:
I've gone ahead and removed the QuestionWrapper entirely as I couldn't think of a good way to support different `onChange` handlers for each question type. I've instead wrapped each question in a `form` manually